src/components/GameTabs.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="games">
-    <PvTabs v-model:active-index="displayGameIndex" scrollable value="0">
+    <PvTabs v-model:active-index="displayGameIndex" scrollable :value="String(displayGameIndex)">
```

Comment:
According to the PrimeVue [docs](https://primevue.org/tabs/) there is no `active-index` model? 

Instead, when using the Tabs in what they call "controlled" mode, we should be using the `value` model: 

```suggestion
    <PvTabs v-model:value="displayGameIndex" scrollable>
```